scala-base/1.0/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM quay.io/continuouspipe/ubuntu:16.04_v1
```

Comment:
@kierenevans & @andytson-inviqa named it to `quay.io/continuouspipe/ubuntu16.04:v1.0`